WIP docs enhancements (#450)
src-d/lookout: docs/lookout-sdk.md

$ lookout-sdk push
```

> `lookout-sdk` will exit with a non `0` exit code if the called analyzer did not responded or if it returned an error.

Reviewer: I think in case the analyzer does not respond, the execution will not end, because we don't have any timeouts.
What about:
```suggestion
`lookout-sdk` will exit with a non `0` exit code if the gRPC call to the analyzer ends with an error.
```

Author: I meant: _if the analyzer could not be called_;
what about this other way?
```suggestion
`lookout-sdk` will exit with a non `0` exit code if the gRPC call to the analyzer
ends with an error or it could not be performed.
```

Reviewer: To me they are the same: ends with an unreachable error code. But ok if you think it's more clear.
